lively.merger/tests/merger-test.js

```diff
@@ -76,107 +77,96 @@ describe('lively.merger >> Merger', () => {
+      expect(await mergeMorphsWithIdsIntoB(morphA.id, morphB.id)).to.equal(morphB);
     });
 
-    it('mergeMorphsIntoA returns a reference to morphA', () => {
-      expect(Merger.mergeMorphsIntoA(morphA, morphB)).to.equal(morphA);
+    it('mergeMorphsIntoA returns a reference to morphA', async () => {
+      expect(await mergeMorphsIntoA(morphA, morphB)).to.equal(morphA);
     });
 
-    it('mergeMorphsIntoB returns a reference to morphB', () => {
-      expect(Merger.mergeMorphsIntoB(morphA, morphB)).to.equal(morphB);
+    it('mergeMorphsIntoB returns a reference to morphB', async () => {
+      expect(await mergeMorphsIntoB(morphA, morphB)).to.equal(morphB);
     });
 
-    it('mergeMorphsWithIdsIntoA alters morphA correctly', () => {
-      Merger.mergeMorphsWithIdsIntoA(morphA.id, morphB.id);
+    it('mergeMorphsWithIdsIntoA alters morphA correctly', async () => {
+      await mergeMorphsWithIdsIntoA(morphA.id, morphB.id);
       expect(morphA.fill).to.equal(Color.red);
       expect(morphA.name).to.equal('name2');
     });
 
-    it('mergeMorphsWithIdsIntoB alters morphB correctly', () => {
-      Merger.mergeMorphsWithIdsIntoB(morphA.id, morphB.id);
+    it('mergeMorphsWithIdsIntoB alters morphB correctly', async () => {
+      await mergeMorphsWithIdsIntoB(morphA.id, morphB.id);
       expect(morphB.fill).to.equal(Color.red);
       expect(morphB.name).to.equal('name2');
     });
 
-    it('mergeMorphsIntoA alters morphA correctly', () => {
-      Merger.mergeMorphsIntoA(morphA, morphB);
-      expect(morphA.fill).to.equal(Color.red);
-      expect(morphA.name).to.equal('name2');
+    it('mergeMorphsIntoA alters morphA correctly', async () => {
+      await mergeMorphsIntoA(morphA.id, morphB.id);
+      expect(morphB.fill).to.equal(Color.red);
+      expect(morphB.name).to.equal('name2');
     });
 
-    it('mergeMorphsIntoB alters morphB correctly', () => {
-      Merger.mergeMorphsIntoB(morphA, morphB);
+    it('mergeMorphsIntoB alters morphB correctly', async () => {
+      await mergeMorphsIntoB(morphA.id, morphB.id);
       expect(morphB.fill).to.equal(Color.red);
       expect(morphB.name).to.equal('name2');
     });
   });
 
   describe('#mergeMorphsWithIds', () => {
-    it('detects if the morphs with the first ID is not alive', () => {
-      expect(() => {
-        Merger.mergeMorphsWithIds('morphAid', 'morphBid');
-      }).to.throw('Cannot merge morphs, morphA with id morphAid not found');
+    it('detects if the morphs with the first ID is not alive', async function () {
+      expect(mergeMorphsWithIds('morphAid', 'morphBid')).to.be.rejectedWith('Cannot merge morphs, morphA with id morphAid not found');
     });
 
-    it('detects if the morphs with the second ID is not alive', () => {
-      expect(() => {
-        Merger.mergeMorphsWithIds(morphA.id, 'morphBid');
-      }).to.throw('Cannot merge morphs, morphB with id morphBid not found');
+    it('detects if the morphs with the second ID is not alive', async () => {
```

Comment:
This doesn't need to be async as well as we use .eventually

```suggestion
    it('detects if the morphs with the second ID is not alive', () => {
```